### PR TITLE
Ui fixes

### PR DESCRIPTION
**Pull request by kkateq**
Katana UI fixes: 
- colors scheme changed
- builders history added to home page
- home page image changed
- paused slave status updated

### CODE REVIEW

File: www/sass/modules/page-elements/_buttons.scss [hunk 1]
```diff
@@ -67,6 +67,16 @@ a.grey-btn,
     }
 }
 
+.grey-btn{
+	@include bgimage(linear-gradient(bottom, $gray-lighter 10%, $gray-light 100%)); 
+		border: 1px solid $gray-lighter;
+		border-bottom:1px solid $gray-light;
+		color: $text-color;
+		&:hover{
+					color: $link-hover-color;
```
Comment (opdude) [inline www/sass/modules/page-elements/_buttons.scss]: A lot of extra spaces here?

File: www/script/project/helpers.js [hunk 2]
```diff
@@ -606,8 +608,53 @@ define(function (require) {
...
+              for (var i = 0; i < ext_history_list.length; i++) {
+                var el = ext_history_list[i];
+                var html = "<div class='row'><div class='col-md-3'><strong>" + unescape(el.proj)
+                          + "</strong></div><div class='col-md-5'><a class='btn btn-default btn-xs' href='" + el.url + "'><img class='btn-icon' src='images/branch.png' alt='branch'/><strong> branch: </strong>" + unescape(el.branch) + "</a></div><div class='col-md-4'><span class='last-run'>"+ moment(el.time).fromNow()+"</span></div></div>";
+                $("<li />", {html: html}).appendTo(hist);
+              }
+            }
+          }
+          else {              
+            var matches = location.href.match(new RegExp(/^.*\/projects\/([^\/]*)\/builders\?unity_branch=(.*)$/));
```
Comment (mariangemarcano) [inline www/script/project/helpers.js]: We need to fix this regex and make it more generic

Comment (opdude) [inline www/script/project/helpers.js]: I would use this regex so we can match with multiple branches etc.

^.*\/projects\/([^\/]*)\/builders\/


Then use codebasesFromURL to get all of the current branches (in the helpers.js file)

File: www/templates/layout.html [hunk 3]
```diff
@@ -6,6 +6,8 @@
     {% block head %}
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href='http://fonts.googleapis.com/css?family=Pacifico&amp;subset=latin' rel='stylesheet' type='text/css'>
```
Comment (opdude) [inline www/templates/layout.html]: Maybe we can grab the full font and add it via the production files? So we have it all locally hosted

**Comment by opdude** [comment]
I'm happy, @mariangemarcano Do you want to merge this?

File: www/script/project/helpers.js [hunk 4]
```diff
@@ -606,8 +608,58 @@ define(function (require) {
...
+                
+                var html = "<div class='row'><div class='col-md-8'><a class='builder-link' href='" + el.url + "'>" + unescape(el.proj) + "</a></div><div class='col-md-4'><span class='last-run'>"+ moment(el.time).fromNow()+"</span></div></div>";
+                var listItem = $("<li />", {html: html});
+                codebasesHtml.appendTo(listItem);
+                listItem.appendTo(hist);
+              }
+            }
+          }
+          else {              
+            var matches = location.href.match(new RegExp(/^.*\/projects\/([^\/]*)\/builders\?(.*)$/));
```
Comment (mariangemarcano) [inline www/script/project/helpers.js]: ? after builders needs to be optional

Comment (opdude) [inline www/script/project/helpers.js]: Is that mainly for us? I can't see many users wanting to see links to pages without branches?

**Pull request merged**